### PR TITLE
WT-6253 Fix time window validation for out of order timestamps

### PR DESCRIPTION
**Pull request by quchenhao**

### CODE REVIEW

File: src/support/timestamp.c [hunk 1]
```diff
@@ -160,7 +160,9 @@ __wt_time_aggregate_validate(
           "aggregate %s",
           __wt_time_aggregate_to_string(ta, time_string[0]));
 
-    if (ta->newest_start_durable_ts > ta->newest_stop_ts)
+    /* In the case of out of order timestamps, start is equal to stop. */
+    if (ta->newest_start_durable_ts != ta->newest_stop_durable_ts &&
+      ta->newest_start_durable_ts > ta->newest_stop_ts)
```
Comment (agorrod) [inline src/support/timestamp.c:165]: Is the distinction between `newest_stop_durable_ts` and `newest_stop_ts` important here? Would `ta->newest_start_durable_ts >= ta->newest_stop_ts` be equivalent?

Comment (quchenhao) [reply src/support/timestamp.c:165]: If everything on the page are out of order, it's possible. But very very unlikely.

Comment (agorrod) [reply src/support/timestamp.c:165]: OK - I think if you need two different comparators, it deserves a more explicit description of why one compares `stop_durable` the other just `stop` - that's not obvious to me.

**Comment by quchenhao** [comment]
The test failure is a known issue.

**Pull request merged**